core/storage/storage.go

```diff
@@ -132,6 +133,9 @@ func DoVolumeRemove(volume model.Volume, storagePool model.StoragePool, progress
 		for i := 0; i < 3; i++ {
 			err := dockerClient.VolumeRemove(context.Background(), volume.Name, false)
 			if err != nil {
+				if strings.Contains(err.Error(), "Should retry") {
+					return errors.Wrap(err, constants.DoVolumeRemoveError+"Normal error. Should return error to cattle and retry")
```

Comment:
OK @StrongMonkey, I'll buy this solution, but the message isn't very good. Lets go with something simpler like "Error removing volume"